tests/scattering1d/test_utils_scattering1d.py

```diff
@@ -55,9 +55,11 @@ def test_border_indices(random_state=42):
 
 # Check that the default frontend is numpy and that errors are correctly launched.
 def test_scattering1d_frontend():
-    scattering = Scattering1D(2, shape=(10, ))
+    scattering = Scattering1D(2, shape=(16, ))
```

Comment:
why?

Reply:
To avoid this warning

```
/Users/user/kymatio/kymatio/scattering1d/filter_bank.py:309: UserWarning: Signal support is too small to avoid border effects
  warnings.warn('Signal support is too small to avoid border effects')
```